packages/scaffolding-cli/templates/test/lighthouse-ci/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "lighthouse-ci",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Run Google Lighthouse for the footer UI",
+  "scripts": {
+    "lh:collect": "lhci collect --config lighthouserc.json",
+    "lh:assert": "lhci assert --config lighthouserc.json",
+    "lh:upload": "lhci upload --config lighthouserc.json"
+  },
```

Comment:
this could be left out and guide the user to run these commands instead, i.e. the task in TFS could look like this

`lhci collect --config path/to/your/lighthouserc.json`
what do you think?